### PR TITLE
Gem theme

### LINKED ISSUE #6: Convert to Gem based theme
Rather than all the content in a site that needs to be cloned as the starting point for a new site, let's instead make this a Gem based theme that we can reuse in various places:

https://jekyllrb.com/docs/themes/

### PR DESCRIPTION
**Pull request by gonsie**
Yay! Gem-based theme that works with GitHub pages 😄 
Closes #6.

### CODE REVIEW

**Comment by gonsie** [comment]
The [MACSio page](http://llnl.github.io/MACSio/) is now using the new remote theme feature (and it works!!!), but using my personal repo. I'll update those settings once this PR is merged.

File: jekyll-llnl-theme.gemspec [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+
+Gem::Specification.new do |spec|
+  spec.name          = "jekyll-llnl-theme"
+  spec.version       = "0.1.0"
+  spec.authors       = ["Elsa Gonsiorowski", "Ian Lee", "Lawrence Livermore National Security, LLC"]
+  spec.email         = ["gonsie@me.com"]
```
Comment (IanLee1521) [inline jekyll-llnl-theme.gemspec:7]: Do we want to use your work email?

File: README.md [hunk 2]
```diff
@@ -1,37 +1,107 @@
...
+```
+
+## Development
+
+### With Ruby Gems
 
 Assuming working on OS X, tested on OS X 10.11.2. `gem` (Ruby package manager)
 was preinstalled, so just working from there:
 
+```shell
```
Comment (IanLee1521) [inline README.md]: If you add that should we outdent the code?

**Review by IanLee1521 — CHANGES_REQUESTED** [review]
Minor changes, but looks good!

File: jekyll-llnl-theme.gemspec (hunk 1 above)
Comment (gonsie) [reply jekyll-llnl-theme.gemspec:7]: Good idea, but no... I use my personal email for most github-y things.

**Pull request merged**